menas/ui/service/MonitoringService.js

```diff
@@ -363,8 +363,7 @@ var MonitoringService = new function() {
 
       if (oData.length > 0) {
         for (let oRun of oData) {
-          oRun["infoDate"] = new Date(oRun["informationDateCasted"]["$date"]); // milliseconds to date
-          oRun["infoDateString"] = Formatters.infoDateToString(oRun["infoDate"]);
+          oRun["infoDateString"] = oRun["controlMeasure"]["metadata"]["additionalInfo"]["enceladus_info_date"]
```

Comment:
Because there is currently no convention on which timezone to use for the infodate I don't see a reliable way to implement this. 

From the other hand, the infodate is mostly treated as a string. So the idea is to "show the user exactly what they entered" and hide the internal transformations. Currently, the cast to ISOdate is only needed to: 1) search the runs 2) sort the runs

Reply:
More exact: Currently we cast infodate string to ISOdate in mongo query for the search, and we assume everything is in UTC (In case the user uses some other timezones she can adjust her search interval accordingly).  Then we also use it to order the results, which works regardless of the timezone (as long as the same one is used across all the dataset runs). That part remained unchanged. Now the change is in what is displayed in the UI:
- Previously, it was casted back to string from the ISOdate. And therefore, prone to the timezone   shift.
- Now we just show the user exactly the string which she entered herself.

Reply:
I am still not convinced correct-correct solution. But as the API will be changing anyway in E3, it can be adjuested then. And until then this is _good enough_ 😉